docs/reference/components/sources/docker.cue

```diff
@@ -212,17 +208,39 @@ components: sources: docker: {
 	]
 
 	how_it_works: {
-		connecting_to_docker: {
-			title: "Connecting To The Docker Daemon"
+		alternatives: {
+			title: "Alternate Strategies"
 			body: #"""
-				Vector will automatically attempt to connect to the docker daemon for you. If
-				the user that Vector is running under can run `docker ps` then Vector will be
-				able to connect. Vector will also respect if `DOCKER_HOST` and
-				`DOCKER_VERIFY_TLS` are set (as well as other Docker environment variables).
-				See the [Docker daemon docs][urls.docker_daemon_socket_option].
+				If you'd prefer not to collect Docker logs through the Docker
+				API Vector offers alternative strategies. In some sets
+
+				First, it's worth mentioning that Vector strives to guide you towards the
+				optimal observability setup without presenting you with unnecessary details or
+				questions. Unfortunately, there are circumstances where trade-offs must be made
+				and you must determine which trade-offs are appropriate. Docker is one of these
+				circumstances.
+
+				Second, if you have a large container-based deployment you should consider using
+				a platform Kubernetes. These platforms provide alternate log collection means
+				that side-step the Docker logging problems. For supported platforms see Vector's
+				[Platforms installation section][docs.installation.platforms].
+
+				Finally, if you cannot use a container orchestrator then you can configure a
```

Comment:
This "ordered list" of paragraphs reads weird to me. I might drop the "Second" and "finally".